### ISSUE TITLE
Item sheets require a lot of clickthru because the notes page is active by default.

### ISSUE THREAD

**Issue by dosuserx** [fvtt-fria-ligan/deathinspace-foundry-vtt#37]
one last issue, the items open up to the notes page of the item instead of data page, which is just an empty page for every item. its really annoying. could you flip that so the default is the data?

**Comment by asherett** [comment]
Agreed. I think both tabs could be joined in one page, split horizontally. Some items (like Heavy EVA Suit) has a full description in the Notes field, but most items have no Notes.
